### ISSUE TITLE
Provide docs for csv-to-pouchdb and maybe a separate package

### ISSUE THREAD

**Issue by ikusteu** [librocco/librocco#27]
**The issue:** Eslint complaining about `csv-to-pouchdb.ts` in `pkg/web`.
I haven't been able to solve it in a clean way (I've ignored it from linting for the time being), so there are a couple points to make it kosher:

1. Does `csv-to-pouchdb.ts` belong in `packages/web` or would it make more sense to move it to a separate package?
2. How should it be consumed, from the browser client, or command line, some other way???
3. Some documentation should be provided on the matter - in form of README.md or comments (I would prefer both)
4. TS could probably be set up to play nicely with deno, but that would make sense to be set up in a separate package: if necessary, there could be two `pacakge.json`s in `package/web` one for deno, and other for JS client (React), but for that we first need to answer the question from point 1.

**Comment by fadwamahmoud** [comment]
1. It doesn't. I created a separate package for it: `scripts` in PR #15 .
2. The whole purpose of it is when an admin (Silvio in our case) has a csv file and wants to insert those records into couchdb, so it's consumed from the command line only for now.
3. Yes. I did provide some comment docs for functions in PR #15 but some README docs are yet to be provided.
4. I think we should do that when we have a deno backend/ business logic, but for now this is just a script that runs from the command line so I think it makes sense for it to be in the `scripts` package. @ikusteu

**Comment by ikusteu** [comment]
Yup, that clears it up pretry nicely. I think the TS could be set up for scripts package (4. Point) and we’re good and kosher 😉
